src/Token.php

```diff
@@ -68,27 +68,37 @@ public function clear()
         $this->driver->set('timeToken', $this->time);
     }
 
-    /**
-     * @param      $keys
-     * @param null $default
-     */
     public function getMultiple($keys, $default = null)
     {
-    }
+        $cache = [];
+        foreach ($keys as $key) {
+            $cache[$key] = $this->get($key, $default);
+        }
 
-    /**
-     * @param      $values
-     * @param null $ttl
-     */
-    public function setMultiple($values, $ttl = null)
+        return $cache;
+    }
+	/**
+	/*Format like: $arrays = [
+            ['key' => 'name','value' => 'SomeName'],
+			.....*/
+			
+	*/
+    public function setMultiple($caches, $ttl = null)
```

Comment:
Argument should be named $values 
like here: https://www.php-fig.org/psr/psr-16/